### PR TITLE
Adding kubernetes-dashboard v1.10.1 deployment to kops addons

### PR DESCRIPTION
**Pull request by schweizerbolzonello**
Adding kubernetes-dashboard v1.10.1 deployment to kops addons
(CVE-2018-18264 + fixes + other updates)

https://github.com/kubernetes/dashboard/releases/tag/v1.10.1

### CODE REVIEW

File: addons/kubernetes-dashboard/addon.yaml [hunk 1]
```diff
@@ -51,3 +51,8 @@ spec:
     selector:
       k8s-addon: kubernetes-dashboard.addons.k8s.io
     manifest: v1.10.0.yaml
```
Comment (mikesplain) [inline addons/kubernetes-dashboard/addon.yaml]: I wonder if it makes sense to remove the old version? Thoughts?

Comment (schweizerbolzonello) [reply addons/kubernetes-dashboard/addon.yaml]: I agree, i think it's more more safe with the CVE
Will update my PR

**Comment by schweizerbolzonello** [comment]
@mikesplain 
Updated PR per your comment on the review

**Comment by chrisz100** [comment]
/lgtm 

Thank you, looks good!

**Comment by mikesplain** [comment]
Oh great! I was just asking the question but I agree, might as well just suggest everyone move to the new version. Thanks @schweizerbolzonello!

/lgtm
/approve

**Pull request merged**